gap/matrix.gi

```diff
@@ -763,6 +767,12 @@ InstallGlobalFunction( FindKernelLowerLeftPGroup,
     return true;
   end );
 
+# computes a straight line program (SLP) for an element <g> of for a p-group
+# in <ri> (that corresponds to a matrix group consisting of lower triangular
```

Comment:
Here a parenthesis opens that never is closed.

Also, is the p-group really "in <ri>"? Perhaps the p-group corresponds to <ri>, or is described by <ri>, or... ?